gdal_subsetter/transform.py

```diff
@@ -1164,14 +1141,14 @@ def subset2(self, tiffile, outputfile, bbox=None, band=None, shapefile=None):
                 bbox - [left,low,right,upper] in lon/lat coordinates
                 shapefile - a shapefile directory in which multiple files exist
         """
-        process_flags["subset"] = True
+        process_flags['subset'] = True
         # covert to absolute path
         tiffile = os.path.abspath(tiffile)
         outputfile = os.path.abspath(outputfile)
-        #RasterFormat = 'GTiff'
+        # RasterFormat = 'GTiff'
```

Comment:
For reference, `pylint` dislikes inline comments that don't have a space after the `#`.